packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -741,6 +741,8 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
     clients: {
       /** Changes to these URLs must also be synched in src/plugins/custom_integrations/server/language_clients/index.ts */
       guide: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/client/index.html`,
+      goConnecting: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/client/go-api/${DOC_LINK_VERSION}/connecting.html`,
```

Comment:
🤔 I wonder if we should go ahead and create serverless specific keys even if we use the existing client doc links as placeholders until we have serverless client docs.